src/window.c

```diff
@@ -190,6 +191,58 @@ int S2D_Show(S2D_Window *window) {
             window->on_mouse(event);
           }
           break;
+        case SDL_FINGERMOTION:
+         {
+            if (window->on_touch) {
+           float fx,dx, fy, dy; 
+           fx = e.tfinger.x * window->width; // TODO: Viewport in scaling
+           fy = e.tfinger.y * window->height;
+           dx = e.tfinger.dx * window->width;
+           dy = e.tfinger.dy * window->height;
+           S2D_GetMouseOnViewport(window, (int)fx, (int)fy, &mx, &my);
```

Comment:
I'm a bit confused by the multiplication of the window dimensions. Does `S2D_GetMouseOnViewport` not correctly translate the points in the window? (Since we're using it for more than mouse now, should probably rename it to `S2D_GetPointOnViewport` or something.)